lib/plan/setting_panel/setting_panel.dart

```diff
@@ -0,0 +1,284 @@
+                if (state.includeBikeSuggestions)
+                  Container(
+                    margin: const EdgeInsets.only(
+                      left: 55,
+                    ),
+                    child: CustomExpansionTile(
+                      title: "Biking speed",
+                      options: const [
+                        '10 Km/h',
+                        '15 Km/h',
+                        '20 Km/h',
+                        '25 Km/h',
+                        '30 Km/h',
+                      ],
+                      textSelected: state.typeBikingSpeed.name,
+                      onChanged: (value) {
+                        switch (value) {
+                          case '10 Km/h':
+                            settingPanelCubit.setBikingSpeed(BikingSpeed.slow);
+                            break;
+                          case '15 Km/h':
+                            settingPanelCubit.setBikingSpeed(BikingSpeed.calm);
+                            break;
+                          case '20 Km/h':
+                            settingPanelCubit.setBikingSpeed(BikingSpeed.average);
+                            break;
+                          case '25 Km/h':
+                            settingPanelCubit.setBikingSpeed(BikingSpeed.prompt);
+                            break;
+                          case '30 Km/h':
+                            settingPanelCubit.setBikingSpeed(BikingSpeed.fast);
+                            break;
+                          default:
+                        }
+                      },
+                    ),
+                  )
+                else
+                  Container(),
+                const Divider(thickness: 2),
+                CustomSwitchTile(
+                  title: 'Park and Ride',
+                  secondary: Icon(TransportMode.bicycle.icon),
+                  value: state.includeParkAndRideSuggestions,
+                  onChanged: settingPanelCubit.setParkRide,
+                ),
+                CustomSwitchTile(
+                  title: 'Car',
+                  secondary: Icon(TransportMode.car.icon),
+                  value: state.includeCarSuggestions,
+                  onChanged: settingPanelCubit.setIncludeCarSuggestions,
+                ),
+                const Divider(thickness: 10),
+                Container(
+                  padding: const EdgeInsets.all(16.0),
+                  child: Text('Accessibility', style: theme.textTheme.bodyText1),
+                ),
+                CustomSwitchTile(
+                  title: 'Wheelchair',
+                  secondary: Icon(TransportMode.car.icon),
+                  value: state.wheelchair,
+                  onChanged: settingPanelCubit.setWheelChair,
+                ),
+                const SizedBox(height: 10),
+              ],
+            );
+          },
+        ),
+      ),
+    );
+  }
+
+  String _getTextWalkingSelect(BuildContext context, WalkingSpeed selected) {
```

Comment:
I would do that with an extension method on the enum.

Reply:
Thanks :) , it is changed